### DISCUSSION TITLE
How to use "_thread" on teensy4.1 development board？

### DISCUSSION THREAD

**Discussion by allan203** [micropython/micropython#14111] [m.iMX RT / Teensy 4.x]
Hello everyone, I am developing something using teensy4.1. How do I use "_thread" on the teensy4.1 development board (I found that this module does not exist on this development board)?
Does anyone know, thank you very much！！！

**Comment by robert-hh** [comment]
Threading is not implemented for the MIMXRT port. There is an open PR to support it, and I have implemented it myself as well, but never made a PR. Until the PR is eventually merged, you would have to build the firmware yourself, of I could build a firmware for you.

**Reply by mattytrentini** [reply]
(See #13755)

**Comment by allan203** [comment]
Thank you so much!
